src/NexusMods.App.UI/Tickers.cs

```diff
@@ -0,0 +1,18 @@
+using R3;
+
+namespace NexusMods.App.UI;
+
+public static class Tickers
+{
+    private static readonly TimeSpan Period = TimeSpan.FromSeconds(30);
+    public static Observable<DateTimeOffset> Primary { get; }
```

Comment:
I would name this based on the period amount, or something vaguely descriptive, like `SlowPoll`